### PR TITLE
Add Group 24 files

### PR DESCRIPTION
**Pull request by To5BG**
Add posters and student/group yaml files.

### CODE REVIEW

File: content/posters/2024/Q4/Group 24/students/Nils Mikk/student_info.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+thesisTitle: 'Program Synthesis from Rewards with Probe: Adjusting Probe to Increase Exploration when Synthesizing Programs From Rewards in Minecraft'
```
Comment (eErr0Re) [inline content/posters/2024/Q4/Group 24/students/Nils Mikk/student_info.yaml:1]: Change `z` to `s` in `Synthesizing`, and `F` to lowercase in `From`.
```suggestion
thesisTitle: 'Program Synthesis from Rewards with Probe: Adjusting Probe to Increase Exploration when Synthesising Programs from Rewards in Minecraft'
```

File: content/posters/2024/Q4/Group 24/students/Nils Mikk/student_info.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+paperUrl: ''
+firstName: Nils
+lastName: Mikk
+repositoryUrl: 'https://github.com/Herb-AI/HerbSearch.jl/tree/probe'
```
Comment (To5BG) [inline content/posters/2024/Q4/Group 24/students/Nils Mikk/student_info.yaml:5]:
```suggestion
repositoryUrl: 'https://github.com/Herb-AI/HerbSearch.jl/tree/probe-with-minerl-explore'
```

File: content/posters/2024/Q4/Group 24/students/Timur Mukminov/student_info.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+thesisTitle: 'Reward Based Program Synthesis for Minecraft: Adapting Program Synthesizers for Reward Evaluation and Leveraging Discovered Programs'
+paperUrl: ''
+firstName: Timur
+lastName: Mukminov
+repositoryUrl: 'https://github.com/Herb-AI/HerbSearch.jl/tree/probe'
```
Comment (To5BG) [inline content/posters/2024/Q4/Group 24/students/Timur Mukminov/student_info.yaml:5]:
```suggestion
repositoryUrl: 'https://github.com/Herb-AI/HerbSearch.jl/tree/probe-with-minerl-exploit'
```

File: content/posters/2024/Q4/Group 24/students/George Latsev/student_info.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+thesisTitle: 'Playing Minecraft with Program Synthesis: Adapting FrAngel to Uncover Diverse Subprograms'
+paperUrl: ''
+firstName: George
+lastName: Latsev
+repositoryUrl: 'https://github.com/Herb-AI/HerbSearch.jl/tree/frangel'
```
Comment (To5BG) [inline content/posters/2024/Q4/Group 24/students/George Latsev/student_info.yaml:5]:
```suggestion
repositoryUrl: 'https://github.com/Herb-AI/HerbSearch.jl/tree/frangel-with-minerl-explore'
```

File: content/posters/2024/Q4/Group 24/students/Alperen Guncan/student_info.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+thesisTitle: 'Program Synthesis from Game Rewards Using FrAngel: Finding Complex Subprograms for Solving Minecraft'
+paperUrl: ''
+firstName: Alperen
+lastName: Guncan
+repositoryUrl: 'https://github.com/Herb-AI/HerbSearch.jl/tree/frangel'
```
Comment (To5BG) [inline content/posters/2024/Q4/Group 24/students/Alperen Guncan/student_info.yaml:5]:
```suggestion
repositoryUrl: 'https://github.com/Herb-AI/HerbSearch.jl/tree/frangel-with-minerl-exploit'
```

File: content/posters/2024/Q4/Group 24/students/Nicolae Filat/student_info.yaml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+thesisTitle: 'Program Synthesis from Rewards using Probe and FrAngel: Impact of Exploration-Exploitation Configurations on Probe and FrAngel in Minecraft'
+paperUrl: ''
+firstName: Nicolae
+lastName: Filat
+repositoryUrl: 'https://github.com/Herb-AI/HerbSearch.jl/tree/dev'
```
Comment (To5BG) [inline content/posters/2024/Q4/Group 24/students/Nicolae Filat/student_info.yaml:5]:
```suggestion
repositoryUrl: 'https://github.com/Herb-AI/HerbSearch.jl/tree/exploration-vs-exploitation-in-frangel-and-probe'
```

**Pull request merged**